<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cameraEditor</class>
 <widget class="QWidget" name="cameraEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QFrame" name="frame">
     <layout class="QVBoxLayout" name="verticalLayout_59">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_51">
        <property name="text">
         <string>Render camera</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="cameraList">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideMiddle</enum>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="uniformRowHeights">
         <bool>true</bool>
        </property>
        <property name="itemsExpandable">
         <bool>false</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <attribute name="headerDefaultSectionSize">
         <number>130</number>
        </attribute>
        <attribute name="headerStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Focal length</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Current</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>9</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3" rowspan="2">
    <widget class="QFrame" name="frFRF">
     <layout class="QVBoxLayout" name="verticalLayout_60">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_63">
        <property name="text">
         <string>Film Response Function</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="frfList">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="uniformItemSizes">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="gbExposure">
     <property name="title">
      <string>Exposure control</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_27">
      <property name="margin">
       <number>3</number>
      </property>
      <item>
       <widget class="QFrame" name="frIso">
        <layout class="QVBoxLayout" name="verticalLayout_56">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_52">
           <property name="text">
            <string>Film ISO</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="filmISO">
           <property name="maximumSize">
            <size>
             <width>108</width>
             <height>16777215</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>20</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>25</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>50</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>64</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>80</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>100</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>125</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>160</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>250</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>320</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>500</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>800</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1000</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1250</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2000</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frShutter">
        <layout class="QVBoxLayout" name="verticalLayout_57">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_61">
           <property name="text">
            <string>Shutter</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="shutter">
           <property name="maximumSize">
            <size>
             <width>110</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frFStop">
        <layout class="QVBoxLayout" name="verticalLayout_58">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_62">
           <property name="text">
            <string>f-stop</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="fstop">
           <property name="maximumSize">
            <size>
             <width>110</width>
             <height>16777215</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>1.4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2.0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2.8</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4.0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5.6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>7.0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8.0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>11.0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>16.0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>22.0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>32.0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>45.0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>64.0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>90.0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>128.0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>180.0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>256.0</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="gbDOF">
     <property name="title">
      <string>DOF</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="frFD">
        <layout class="QGridLayout" name="gridLayout">
         <property name="horizontalSpacing">
          <number>2</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_65">
           <property name="text">
            <string>Focal distance</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="focalDistance">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>meters</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frFStop2">
        <layout class="QVBoxLayout" name="verticalLayout_61">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_64">
           <property name="text">
            <string>f-stop</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="dofStop">
           <property name="maximumSize">
            <size>
             <width>110</width>
             <height>16777215</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>1.4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2.0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2.8</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4.0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5.6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>7.0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8.0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>11.0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>16.0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>22.0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>32.0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>45.0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>64.0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>90.0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>128.0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>180.0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>256.0</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
